boreas/ping.c

```diff
@@ -253,31 +254,44 @@ send_icmp (gpointer key, gpointer value, gpointer scanner_p)
   struct in_addr dst4;
   struct in_addr *dst4_p = &dst4;
   static int count = 0;
+  int icmp_retries, grace_period = 0;
+  const char *tmp;
+  if ((icmp_retries =
+         (tmp = prefs_get ("ICMP_RETRIES")) != NULL ? atoi (tmp) : 1)
```

Comment:
```suggestion
         (tmp = prefs_get ("icmp_retries")) != NULL ? atoi (tmp) : 1)
```
All prefs in man page a lower case.